js/src/vector.ts

```diff
@@ -414,6 +414,16 @@ export class DictionaryVector<T extends DataType = DataType> extends Vector<Dict
     }
     public getKey(index: number) { return this.indicies.get(index); }
     public getValue(key: number) { return this.dictionary.get(key); }
+    public reverseLookup(value: T): number|undefined {
```

Comment:
@trxcllnt do you think it makes sense to implement `reverseLookup` here, or should it be in the view?

Reply:
@TheNeuralBit I think this is in the right place, since it's specific to DictionaryVectors. The View interface is intended to provide a minimal common interface for views over the the Vector buffer layouts (aka the Vector Data types), somewhat analogously to the DataView/TypedArray relationship with ArrayBuffer.